static/app/components/actions/resolve.tsx

```diff
@@ -195,13 +194,10 @@ function ResolveActions({
       },
       {
         key: 'current-release',
-        label:
-          hasIssueReleaseSemver || !latestRelease
-            ? t('The current release')
-            : t('The current release (%s)', formatVersion(latestRelease.version)),
```

Comment:
Why do we no longer include the name of the release next to "current release?"

Reply:
its on the next line instead of the same line